Frontend/src/Pages/PersonalAccount/PersonalReviewsTab/styles/search.module.scss

```
@@ -0,0 +1,37 @@
+.searchFilterBox {
+  .searchForm {
+    padding: 1em;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .searchInputWrapper{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    
+    .searchInput {
+      all: unset;
+      background: #414141;
+      border: none;
+      height: 2em;
+      padding: 1em;
+      border-radius: 0.5em;
+      margin: 0 0.5em;
+      width: 50vw;
```

```suggestion
      width: 100%;
```
тогда не будет скролл бара вправо( он был если экран приближался )